index.js

```diff
@@ -204,50 +234,78 @@ if (commander.init) {
+        const text = `${beginningText}${formattedChangelogs}${endText}`;
+
+        fs.writeFile(paths.changelog, text, (error) => { 
             if (error) throw error; 
         });
-        if (!commander.silent) console.log(`${data}\nappended in /CHANGELOG.md`);
-    }).then(() => {
+        if (!silent) console.log(`${formattedChangelogs}\nappended in /CHANGELOG.md`);
+
         // delete JSON changelog files
-        fs.readdirSync(fileDir).forEach((file) => {
-            fs.access(`${fileDir}/${file}`, error => {
+        fs.readdirSync(paths.unrealeasedChangelogsDir).forEach((file) => {
+            fs.access(`${paths.unrealeasedChangelogsDir}${file}`, error => {
                 if (!error) {
-                    fs.unlinkSync(`${fileDir}/${file}`, (error) => { throw error });
+                    fs.unlinkSync(`${paths.unrealeasedChangelogsDir}/${file}`, (error) => { throw error });
                 } else {
                     throw error;
                 }
             });
         });
-    }).then(() => {
+
         if (config.autoCommitRelease) {
-            const message = config.changelogMessageRelease
-                ? config.changelogMessageRelease.replace(/BRANCH/g, branchNumber)
-                : "changelog";
-            return git().silent(true)
-                .add([`${fileDir}/*`, "CHANGELOG.md", `${process.cwd()}/changelogs/config.json`])
-                .then(() => git().commit(message)) // could not chain git.add().commit() for unknown reason
-                .then(() => {
-                    if (!commander.silent) console.log("Changelog committed, use \`git push\` to write it remotely");
-                });
+            const message = config.changelogMessageRelease || "changelog";
+            await git().silent(true).add([paths.unrealeasedChangelogsDir, paths.changelog, paths.userConfig]);
+            await git().silent(true).commit(message);
+            if (!silent) console.log("Changelog committed, use \`git push\` to write it remotely");
         }
-    }).catch((error) => {
+    } catch (error) {
         console.log(error);
         process.exit();
-    });
-    
-} else {
-    commander.help();
-}
\ No newline at end of file
+    }
+};
+
+async function main() {
+
+    commander
+        .command("init")
+        .description("copy default config into package")
+        .action(init);
+    commander
+        .command("add")
+        .description("write the current git branch changelog file")
+        .option("-m, --message <string>", "provide changelog message")
+        .option("-t, --type <string>", "provide chagelog type (e.g. \"Added\"")
+        .option("-b, --branch <number>", "provide the branch number")
+        .option("-s, --silent", "run in silent mode using default parameters. Uses current branch number, need to pass message and type")
+        .action(add);
+    commander
+        .command("release")
+        .description("concat changelogs file into CHANGELOG.md")
```

Comment:
If I understand this step correctly, it concatenates some files into a single one. Thus here it would be `"concat changelog files into CHANGELOG.md"` rather than `"concat changelogs file into CHANGELOG.md"`.

Am I right?